libraries/RC_Channel/RC_Channels.cpp

```diff
@@ -78,6 +79,11 @@ bool RC_Channels::read_input(void)
 
     last_update_ms = AP_HAL::millis();
 
+    // clear RC override by the pilot input if RC_OPTION is configured
+    if (has_active_overrides()) {
```

Comment:
you could move this "if (has_active_overrides()) {" into clear_overrides_by_rc() although it's not a blocker.

Reply:
fixed